docs/src/main/sphinx/admin/web-interface.rst

```diff
@@ -16,27 +16,28 @@ Authentication
 
 The Web UI requires users to authenticate. If Trino is not configured to require
 authentication, then any username can be used, and no password is required or
-allowed. Typically, users should login with the same username that they use for
+allowed. Typically, users login with the same username that they use for
 running queries.
 
-Accessing the Web UI over HTTPS requires configuring an authentication type for
-the Web UI or the Trino server. If no authentication type is configured for the
-Web UI, then it will chosen based on the Trino server authentication type.
+Accessing the Web UI over HTTPS requires configuring :doc:`TLS support
+</security/https>` for the Trino server, or independently configuring an
+authentication type for the Web UI. If no authentication type is configured for
+the Web UI, then it is chosen based on the Trino server authentication type.
```

Comment:
I'd drop this paragraph entirely.  When these docs were written the WebUI on HTTPS required authentication and that is no longer true, and generally just works, so I would not call out anything special here as it is now more confusing than helpful

Reply:
done